WRP-19034: Fix spotlight.focusElement to check cursor visibility and touch mode (#3159)
enactjs/enact: CHANGELOG.md

The following is a curated list of changes in the Enact project, newest changes on the top.

## [unreleased]

### Fixed

> - `spotlight` to show the focus effect when non-touch pointer mode is changed to `false` while an app is loading

Reviewer: Could you please fix word `non-touch` to be more specific?
```suggestion
- `spotlight` to show the focus effect when the pointer type is not `touch` and pointer mode is changed to `false` while an app is loading
```
